### DISCUSSION TITLE
Defense Bot Command Crashing Issue

### DISCUSSION THREAD

**Discussion by Rat-Crusher** [FakeFishGames/Barotrauma#15026] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I created a defense bot and then proceeded to shift-middleclick it to attempt to give it a command. I've never used the bot until now so I assumed I needed to command it. Doing so crashed the game instantly.

### Reproduction steps

1. Get a defense bot
2. Shift-Middleclick it
3. Game crashes instantly

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in the next unstable build and the next public release. Sorry for the inconvenience!
